content/ja/handbook/company/culture/all-remote/asynchronous.md

```
@@ -0,0 +1,592 @@
+### 外部候補者の面接
+
+GitLab のすべての[面接プロセス](/handbook/hiring/interviewing)には何らかの形の同期コミュニケーションが含まれます。一部のチームは[面接プロセス中に非同期の慣行を活用](https://www.youtube.com/watch?v=jSbCt8b_4ug)していますが、これはすべての面接プロセスで標準的なアプローチではありません。
+
+## 会社外での非同期ワーク
+
+チーム内で非同期ワークを確立していても、会社外の人と作業する際に非同期の慣行を奨励することは困難または不快に感じることがあります。
+
+すべての組織はそれぞれの規範を持っていますが、情報提供と教育を通じて丁寧に現状に挑戦することができます。外部の関係者と非同期で作業するためのアプローチを以下に示します：
+
+1. **同期で始める：** プロジェクトやパートナーシップのキックオフのために同期通話を行ってください。その通話中に、今後のプロジェクトに一部の非同期ワークを組み込みたいことを伝えてください。この通話を[関係とラポールを構築する](asynchronous/#when-to-start-synchronously)ために使用すると、非同期ワークがより効率的になります。
+1. **期待を設定し、行動でモデルを示す：** どのように非同期で一緒に作業するか（どの[ツールを使用するか](#use-the-right-tools)、何を文書化するか、どのくらいの頻度で）を事前に議論してください。将来の同期のためのカデンスに合意することで、全員が何を期待するかがわかります。プロジェクトやパートナーシップを通じてこの行動をモデルにしてください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)